components/Form.js

```diff
@@ -50,8 +51,10 @@ export default function Form({
   const router = useRouter();
   const [enteredTitle, setEnteredTitle] = useState("");
   const [isValid, setIsValid] = useState(false);
-  const [allocatedMembers, setAllocatedMembers] =
-    useState(allocatedMembersList);
+  const [allocatedMembers, setAllocatedMembers] = useState(
+    allocatedMembersList || familyMembers
+  );
+  const [assignedTo, setAssignedTo] = useState(value?.assignedTo || []);
```

Comment:
i think this state could be renamed to represent an array within the naming :) 
```suggestion
  const [assignedMembers, setAssignedMembers] = useState(value?.assignedTo || []);
```

Reply:
Hi Merle, thank you for the review and the suggestion. This state has the same name as a key from the task objects in the tasks list (which were provided to us), because the name of the field in form data is taken from the state by creating/editing a task. If I rename the state, then I need to rename assignedTo everywhere in the app. Otherwise, it doesn't make much sense.